samples/clipboard/clipboard.cpp

```diff
@@ -192,6 +192,7 @@ void MyFrame::OnUpdateUI(wxUpdateUIEvent&event)
 void MyFrame::OnQuit(wxCommandEvent& WXUNUSED(event))
 {
     // true is to force the frame to close
+    wxTheClipboard->Flush();
```

Comment:
This line should be above the comment...

Reply:
@obfuscated ,
Thank you for the review.
I will certainly fix this and while doing so will probably put it on the branch.

One question though - does it work for you? Can this PR mark those tickets in question as solved?

I tested with GTK+3 on Gentoo, but if you can confirm that it works on other systems, it would be great.

Thank you.

Reply:
I've not tried it, I just looked at the code just to see what would change. Also I'm running Gentoo, so it won't improve the config test coverage.